neofs-local/docker-compose.yml

```diff
@@ -6,15 +6,16 @@ networks:
 
 services:
   gw-backend:
-    image: nspccdev/neofs-http-gate:${NEOFS_IMAGES_VERSION}
+    image: nspccdev/neofs-http-gate:${NEOFS_GW_VERSION}
     container_name: neofs-gw-backend
     depends_on:
       storage01:
         condition: service_healthy
     environment:
       - GW_REQUEST_TIMEOUT=40h
```

Comment:
it's too long, I think, that 5-10m will be enough, what you think?

Reply:
Yeah, seems so. I will change timeout duration in next commit.